clang/lib/Format/UnwrappedLineParser.h

```diff
@@ -420,6 +420,8 @@ struct UnwrappedLineNode {
   SmallVector<UnwrappedLine, 0> Children;
 };
 
+std::ostream &operator<<(std::ostream &Stream, const UnwrappedLine &Line);
+
```

Comment:
Delete.

Reply:
Ah, thanks for the explanation.

Reply:
I can move the operator<< to the test header, but I'd then at least need some public method to do the printing.